ports/esp32s2/boards/adafruit_magtag_2.9_grayscale/pins.c

```diff
@@ -16,11 +16,16 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_EPD_CS), MP_ROM_PTR(&pin_GPIO8) },
 
     { MP_ROM_QSTR(MP_QSTR_BUTTON_A), MP_ROM_PTR(&pin_GPIO15) },
+    { MP_ROM_QSTR(MP_QSTR_D15), MP_ROM_PTR(&pin_GPIO15) },
```

Comment:
As long as you're editing to fix D1/D4, could you put in blank lines to group the aliases together?

Reply:
Yep!